<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="text">
    <string>车票预订</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>861</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>出发地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>北京</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>天津</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>上海</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>重庆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>石家庄</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>太原</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>沈阳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>长春</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>哈尔滨</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>南京</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>杭州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>合肥</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>福州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>南昌</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>济南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>郑州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>武汉</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>长沙</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>广州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>海口</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>成都</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>贵阳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>昆明</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西安</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>兰州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西宁</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>拉萨</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>南宁</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>呼和浩特</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>银川</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>乌鲁木齐</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>目的地</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <item>
         <property name="text">
          <string>北京</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>天津</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>上海</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>重庆</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>石家庄</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>太原</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>沈阳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>长春</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>哈尔滨</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>南京</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>杭州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>合肥</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>福州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>南昌</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>济南</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>郑州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>武汉</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>长沙</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>广州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>海口</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>成都</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>贵阳</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>昆明</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西安</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>兰州</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>西宁</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>拉萨</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>南宁</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>呼和浩特</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>银川</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>乌鲁木齐</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>861</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>出发日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="currentText">
         <string>20180709</string>
        </property>
        <item>
         <property name="text">
          <string>20180709</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180710</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180711</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180712</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180713</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180714</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180715</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180716</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180717</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180718</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180719</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180721</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180722</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180723</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180724</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180725</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20180726</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>出发时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_4">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>不限</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>00:01-08:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>08:01-12:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12:01-16:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16:01-20:00</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20:01-00:00</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>861</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>40</pointsize>
         </font>
        </property>
        <property name="text">
         <string>列车类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_5">
        <item>
         <property name="text">
          <string>不限</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>高铁/动车组 G/D/C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>普速列车 Y/K/T/Z</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>学生</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>20</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
